.gitlab-ci.yml

```diff
@@ -81,6 +81,9 @@ gcp-dev:
     # Clean up even if something failed.
     - cd gcp/live/dev
     - rake destroy
+    # Forcefully repopulate secrets to exercise managing secrets from scratch.
```

Comment:
Maybe "Destroy secrets, so they can be repopulated on next run to exercise managing secrets from scratch?"